woocommerce/woocommerce-android
# Free Trial: Adjust free trial banner UI

Summary
==========
Fix issues #8590, #8588 and #8587 by introducing a series of adjustments to the Free Trial banner UI component and the `TrialStatusBarFormatter` class.

Screenshots
==========
| Scenario | Now | Before |
| ------------- | ------------- | ------------- |
| Dark Mode | ![Screenshot_20230323_010645](https://user-images.githubusercontent.com/5920403/227100857-1fb3a725-3931-4d0b-973d-46722dc231a5.png) | ![Screenshot_20230323_011241](https://user-images.githubusercontent.com/5920403/227101242-0ce705d0-aaf1-4637-9283-d638338d8ff7.png) |
| Day count | ![Screenshot_20230323_010734](https://user-images.githubusercontent.com/5920403/227100868-3db9a76c-3152-45d1-85f8-2f8cddfea01d.png) | ![Screenshot_20230323_011343](https://user-images.githubusercontent.com/5920403/227101286-2105c37e-81a8-441d-8002-e9270611ad12.png) |
| Font size | ![Screenshot_20230323_010831](https://user-images.githubusercontent.com/5920403/227100884-2ef27c3a-d73e-4ab8-86e2-b9dab73d664e.png) | ![Screenshot_20230323_011206](https://user-images.githubusercontent.com/5920403/227101261-91557f65-1cdb-46fc-86e4-4463101d1b9c.png) |

How to Test
==========
1. Open the app with a running Free Trial site with one day remaining
2. Set the app in dark mode and configure the Android system to use the biggest font size available
3. Verify that the Free Trial banner shows the message as `1 day` correctly, with the expected dark mode colors and the text size doesn't affect the banner text visibility at all

Update release notes:
